std/zip.d

```diff
@@ -435,6 +437,79 @@ private:
     enum zip64EndOfCentralDirLocatorLength = 20;
     enum dataDescriptorLength = 12;
 
+    align (1) // just to make sure there are no gaps
```

Comment:
`align` must be declared inside the struct to affect it's members (according to https://dlang.org/spec/attribute.html#align)